src/lib/free-queue/test.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>FreeQueueSAB Tests</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/mocha/10.6.0/mocha.min.css" integrity="sha384-BBZj3VfQ5pAok2bd0uXf+8cRAJ3D/JZWrxBlnWx6BcsBO96ZzZR40fpC20MtWbM8" crossorigin="anonymous">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/mocha/10.6.0/mocha.min.js" integrity="sha384-A5EqeN0zznOH0u6V4OduoPlHhkhk+Km4eBBNBq0UAE1Z51cFJfX/f9biOo2SmCCS" crossorigin="anonymous"></script>
```

Comment:
Same issue with the hash as above.

Reply:
Yes, I used an external hash generator site. But now I updated it from the website itself.

Reply:
Thank you, it's working for me now.